packages/site/src/content/AnimatedPresence/index.tsx

```diff
@@ -1,26 +1,20 @@
-import {
-  AnimatedPresence as AnimatedPresenceRoot,
-  Button,
-  Content,
-  Flex,
-  Text,
-} from "@jobber/components";
 import AnimatedPresenceContent from "@atlantis/docs/components/AnimatedPresence/AnimatedPresence.stories.mdx";
-import { PropsWithChildren, useState } from "react";
 import Props from "./AnimatedPresence.props.json";
 import { ContentExport } from "../../types/content";
 import { getStorybookUrl } from "../../layout/getStorybookUrl";
 
-export const AnimatedPresence = (props: PropsWithChildren) => {
-  const [switched, setSwitched] = useState(false);
-
+export default {
+  content: () => <AnimatedPresenceContent />,
+  props: Props,
+  component: {
+    element: `const [switched, setSwitched] = useState(false);
```

Comment:
Ultimately the format is now Strings. Pass your code through as a string. It will be automatically wrapped in a function. If you don't provide a return statement, I'll insert one.

This is to keep the code examples as clean as possible. I'll document how it all works properly before we go live.